modules/gitops-release-notes-1-9-3.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-9-3_{context}"]
+= Release notes for {gitops-title} 1.9.3
+
+{gitops-title} 1.9.3 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-9-3_{context}"]
+== Errata updates
+
+[id="gitops-1-9-3-security-update-advisory_{context}"]
+=== RHSA-2023:XXXX - {gitops-title} 1.9.3 security update advisory
+
+Issued: 2023-11-10
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:XXXX[RHSA-2023:XXXX]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
```

Comment:
Recommend that you put the informative intro phrase before the action in this sentence, so reword as: "If you have installed the {gitops-title} Operator in the default namespace, to view the container images in this release, run the following command:"

Reply:
Addressed. Thank you.